ui/asset-manifest.json

```diff
@@ -1,11 +1,11 @@
 {
-  "main.css": "main.6083c0f7.css",
-  "main.js": "static/js/main.a14c6226.js",
-  "static/js/0.181fa87d.chunk.js": "static/js/0.181fa87d.chunk.js",
-  "static/js/1.b707500e.chunk.js": "static/js/1.b707500e.chunk.js",
-  "static/js/2.50fa135e.chunk.js": "static/js/2.50fa135e.chunk.js",
-  "static/js/3.3d01620c.chunk.js": "static/js/3.3d01620c.chunk.js",
-  "static/js/4.39377f57.chunk.js": "static/js/4.39377f57.chunk.js",
-  "static/js/5.b9434839.chunk.js": "static/js/5.b9434839.chunk.js",
-  "static/js/6.a7f82c8e.chunk.js": "static/js/6.a7f82c8e.chunk.js"
+  "main.css": "main.e75112a3.css",
```

Comment:
is there a better way to do so?

Reply:
These are statically built files from the bodhi-ui repo. the `bodhi-ui/build` folder is copied to `bodhi-electron/ui`.

I will be packaging `bodhi-ui/build` as a node module as well. I just updated it the old way this time so it would run correctly (it has the change of the port number).

Reply:
I see okok